picard/album.py

```diff
@@ -77,6 +85,14 @@ def iterfiles(self, save=False):
             for file in self.unmatched_files.iterfiles():
                 yield file
 
+    def add_album_artist(self, id, name, sort_name):
+        album_artist = AlbumArtist(id, name, sort_name)
+        self._album_artists.append(album_artist)
+        return album_artist
```

Comment:
Since it is convenient for it to return the new AlbumArtist, it could perhaps be renamed 'new_album_artist', or even 'new_artist' (since it is in Album class).
Any proposal ?